ci/jjb/jobs/pulp-backup.yaml

```diff
@@ -0,0 +1,47 @@
+- job-template:
+    name: 'pulp-{pulp_version}-{pulp_build}-backup-{os}'
+    node: '{os}-np'
+    properties:
+        - qe-ownership
+        - build-discarder:
+            days-to-keep: 3
+            num-to-keep: 3
+            artifact-days-to-keep: 3
+            artifact-num-to-keep: 12
+    scm:
+        - pulp-packaging-github
+    wrappers:
+        - config-file-provider:
+            files:
+                - file-id: rhn_credentials
+                  variable: RHN_CREDENTIALS
+        - inject:
+            properties-content: |
+                OS={os}
+                PULP_BUILD={pulp_build}
+                PULP_VERSION={pulp_version}
+    triggers:
+        - timed: '@midnight'
+    builders:
+        - shell:
+            !include-raw-escape:
+                - 'ssh-setup.sh'
+        - shell:
+            !include-raw-escape:
+                - 'pulp-backup.sh'
+        - shell: |
+            if [[ "${{OS}}" =~ "rhel" ]]; then
+                sudo subscription-manager unregister
+                sudo subscription-manager clean
+            fi
+        - capture-logs
+    publishers:
+      - archive:
+          artifacts: '*.bz2'
+      - join-trigger:
+          projects:
+            - 'pulp-2.13-stable-restore-rhel7'
```

Comment:
Since you are using a job template, I think it would be better to create a variable for the pulp version which the backup should be restored, something like restore_pulp_version so you can control on the permutation which should be the triggered job. For example you may have 2.14 version to restore a backup from 2.12 and in the current definition you can't control that.